lib/mix/tasks/gh_pages_docs.ex

```diff
@@ -10,36 +10,82 @@ defmodule Mix.Tasks.GhPagesDocs do
   @shortdoc "Update a documentation version on GH Pages."
 
   @spec run(term) :: :ok
-  def run(_args) do
-    version = "v#{Project.config()[:version]}"
-
-    case File.read!("assets/js/versions.js") |> String.contains?(version) do
-      true ->
-        Mix.Task.run("docs")
-        File.cd!("doc")
-        Mix.shell().cmd("git stash")
-
-        Mix.shell().cmd("git checkout gh-pages")
-
-        case File.mkdir("../#{version}") do
-          result when result in [:ok, {:error, :eexist}] ->
-            Mix.shell().cmd("cp -r ./* ../#{version}")
-            File.cd!("..")
-            Mix.shell().cmd("ls")
-            Mix.shell().cmd("git add #{version}/*")
-            Mix.shell().cmd("git commit -m \"Update #{version}\"")
-            Mix.shell().cmd("rm -rf doc")
-            Mix.shell().cmd("git push origin gh-pages")
-
-          _ ->
-            Mix.raise("Cannot create the #{version} directory")
+  def run([version]) do
+    unless File.read!("assets/js/versions.js") |> String.contains?(version) do
```

Comment:
In one of the other PR's review, I noticed that we try to avoid the pipeline operator in case it's used only once. Do we want to keep the convention here as well?